docs/tests/__snapshots__/props-table.test.ts.snap

```diff
@@ -90,6 +83,13 @@ exports[`Props Table 1`] = `
                         \\"category\\": \\"AlertProps\\",
                         \\"isOptional\\": true
                     },
+                    \\"heading\\": {
```

Comment:
Bumping `devDependencies` have changed the order of this prop for some reason... but the content is not changed.